flake.nix

```
@@ -147,6 +145,16 @@
             # SITE_STYLES skipped by design.
             export NIX_PILLS_MANUAL_IN="${nixPills}/share/doc/nix-pills"
             export NIX_DEV_MANUAL_IN="${nix-dev.defaultPackage.x86_64-linux}/html"
+
+            echo ""
+            echo "  To start developing run:"
+            echo "      python run.py"
+            echo ""
+            echo "  and open browser on:"
+            echo "      https://127.0.0.1:8000"
```

```suggestion
            echo "  and go to the following URL in your browser:"
            echo "      https://127.0.0.1:8000/"
```

The  `/` at the end is me being a purist about URLs, we can drop it. If we do, also drop it in that other comment about `index.html`. (Or use `/index.html` here too.)